tests/api/adbc/test_protocol.py

```diff
@@ -24,27 +52,30 @@ def test_serialize_query_params_complete_query() -> None:
             "metrics": ["a", "b"],
```

Comment:
Do we have any tests against object syntax through adbc? Something like this should work
```
{{ semantic_layer.query(metrics=[Metric("orders")]) }}
```

Reply:
Oh the SDK onlt generates object syntax for order by. For metrics and group by we just use regular string arrays.

Is there a reason we would need that?

Reply:
For order by there are tests tho

Reply:
For extra context: there's no way for a user to submit a raw string to be sent via ADBC. They have to do like:
```python
with client.session():
    table = client.query(metrics=["my_metric"], group_by=["my_dim"])
    print(table)
```

Under the hood, the SDK will generate a SQL statement like the following and send it via ADBC.
```
SELECT * FROM {{ semantic_layer.query(metrics=["my_metric"], group_by=["my_dim"]) }}
```